### DISCUSSION TITLE
Kommunikation gestört, keine neuen Geräte, aber boosten....

### DISCUSSION THREAD

**Discussion by Mig287** [OpenCCU/OpenCCU#2043] [🇩🇪 Fragen&Antworten] [answered]
Hallo Zusammne!

Ich nutze einen RaspB 4 mit HomeAssistant habe dort den HMIP-RFUSB angeschlossen (nicht am USB3 Port) und RapsberryMatic installiert. 
Ich kann aber keine Geräte anlernen (habe 2 Thermostate resettet). Dann habe ich das Backup meiner alten CCU3 eingespielt, mit angerlernten Geräten, diese gehen (mehr oder weniger erwartungsgemäß) auf Kommunikation gestört Ich kann den Boost Befehl durchschicken, aber nichts anderes, bekomme auch nicht die aktuell eingestellte Temperatur zurück.
Habe 2 verschiedene potente 3A Netzteile ausprobiert und den Stick an eine Verlängerungsschnur gepackt. 
Carrier Sense ist meistens bei 0% manchmal bei 1% wobei ich die refresh rate nicht kenne. 
Logs auf Anforderung, weiß nicht welche sinnvoll sind. 

Ich bitte dringend um Hilfe habe mir bereits die letzten 24 Stunden den Kopf zerbrochen und Dinge ausprobiert bin mit meinem Latein am Ende. 

BG
Michael

**Comment by jens-maus** [comment, marked as answer]
Auch dieses mal wird es am RaspberryPi4 bzw. an dessen starken Funkstörungen liegen. Auch wenn der CarrierSense ggf. mit 0-1% ausgespuckt wird tippe ich stark darauf. Also entweder mit RaspberryPi3 probieren oder den HmIP-RFUSB mit langem USB verländerungskabel (2-3m) weit weg vom RaspberryPi betreiben.

**Reply by Mig287** [reply]
Hallo Jens, 

erstmal vielen Dank für die schnelle Antwort! 
Ich bin direkt los zum hiesigen Elektroriesen, das Beste was zu kriegen war ist ein 3m USB A-A, 2.0 Verlängerungskabel mit 0,5A max. Leistung. 
Aber die Problematik ist exakt gleich geblieben. 
Habe natürlich einen RaspberryMatic neustart gemacht. 
edit: Tatsächlich habe ich sogar noch ein 10 Meter Kabel gehabt, wobei ich dort die genauen Specs nicht kenne, und auch damit bleibt die Thematik genau gleich. 
Habe auch nochmal ein anderes Thermostat genommen. 
edit1: Auch nach neu Installation von RspberryMatic Addon können keine Geräte angelernt werden.

**Reply by Mig287** [reply]
Hier mal das HomeAssistant log mit frischer Installation und 10 Meter USB Verlängerung:
Mounting /data as /usr/local (Home Assistant Add-On): OK
Starting watchdog...
Identifying host system: oci, OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.2, OK
Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:01:00.0-1.3.1, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:01:00.0-1.3.1, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: creating new SSL cert... OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: .......OK
Starting rfd: .OK
Starting HMIPServer: .............OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: OK
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.65.11.20221005 (raspmatic_oci_arm64)

**Reply by Mig287** [reply]
Und das RaspberryMatic log:

***** messages *****
Nov 11 16:21:55 de838cd8-raspberrymatic syslog.info syslogd started: BusyBox v1.35.0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (9) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (11) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ kvm guest vtimer(12) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (14) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (17) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ DMA IRQ(20) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ VCHIQ doorbell(27) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ mmc1, mmc0(28) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (29) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (30) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (31) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (32) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (35) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (36) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (41) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic daemon.debug /usr/sbin/irqbalance: IRQ (43) guessed as class 0
Nov 11 16:21:55 de838cd8-raspberrymatic user.info firewall: configuration set
Nov 11 16:21:59 de838cd8-raspberrymatic daemon.err xinetd[951]: Unable to read included directory: /etc/config/xinetd.d [file=/etc/xinetd.conf] [line=14]
Nov 11 16:21:59 de838cd8-raspberrymatic daemon.crit xinetd[951]: 951 {init_services} no services. Exiting...
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: original file not exists, so try load new file. [LoadOM():iseDOM.cpp:2610]
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: new file not exists, so try load bak file. [LoadOM():iseDOM.cpp:2622]
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: failed open file= /etc/config/homematic.regadom.bak [LoadFromFile():iseDOMpersist.cpp:78]
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: (202) failed! Already in map! [Insert():iseDOMobj.cpp:526]
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: IseAddFavorite failed! ID=202 [IseAddFavorite():iseDOM.cpp:644]
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: (203) failed! Already in map! [Insert():iseDOMobj.cpp:526]
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: IseAddFavorite failed! ID=203 [IseAddFavorite():iseDOM.cpp:644]
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: (204) failed! Already in map! [Insert():iseDOMobj.cpp:526]
Nov 11 16:22:32 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: IseAddFavorite failed! ID=204 [IseAddFavorite():iseDOM.cpp:644]
Nov 11 16:22:34 de838cd8-raspberrymatic user.info homematic: meine-homematic.de Versions - Upgrade auf 1.503
Nov 11 16:22:34 de838cd8-raspberrymatic user.info homematic: meine-homematic.de VPN Dienst und Autoupdate DEAKTIVIERT
Nov 11 16:22:36 de838cd8-raspberrymatic daemon.info : starting pid 1328, tty '': '/bin/mv /tmp/boot.log /var/log/boot.log'
Nov 11 16:22:36 de838cd8-raspberrymatic daemon.info : starting pid 1329, tty '/dev/null': '/usr/bin/monit -Ic /etc/monitrc'
Nov 11 16:22:36 de838cd8-raspberrymatic user.info monit[1329]:  New Monit id: 5f94254a6de634e688129375674909d4  Stored in '/usr/local/etc/monit.id'
Nov 11 16:22:36 de838cd8-raspberrymatic user.info monit[1329]: Starting Monit 5.30.0 daemon with http interface at /var/run/monit.sock
Nov 11 16:22:36 de838cd8-raspberrymatic user.info monit[1329]: 'de838cd8-raspberrymatic' Monit 5.30.0 started
Nov 11 16:22:52 de838cd8-raspberrymatic user.err monit[1329]: 'udevdEnabled' status failed (0) -- no output
Nov 11 16:22:53 de838cd8-raspberrymatic user.err monit[1329]: 'sshdEnabled' status failed (1) -- no output
Nov 11 16:22:53 de838cd8-raspberrymatic user.err monit[1329]: 'hs485dEnabled' status failed (2) -- grep: /var/etc/hs485d.conf: No such file or directory
Nov 11 16:22:54 de838cd8-raspberrymatic user.err monit[1329]: 'hmlangwEnabled' status failed (1) -- no output
Nov 11 16:22:54 de838cd8-raspberrymatic user.warn monit[1329]: 'hasUSB' status failed (1) -- no output
Nov 11 16:22:54 de838cd8-raspberrymatic user.err monit[1329]: 'hb_rf_eth-CheckEnabled' status failed (1) -- no output
Nov 11 16:22:54 de838cd8-raspberrymatic user.warn monit[1329]: 'rpi4usb3Check' status failed (1) -- no output
Nov 11 16:22:55 de838cd8-raspberrymatic user.err monit[1329]: 'tailscaleEnabled' status failed (1) -- no output
Nov 11 16:22:55 de838cd8-raspberrymatic user.err monit[1329]: Lookup for '/media/usb1' filesystem failed  -- not found in /proc/self/mounts
Nov 11 16:22:55 de838cd8-raspberrymatic user.err monit[1329]: Filesystem '/media/usb1' not mounted
Nov 11 16:22:55 de838cd8-raspberrymatic user.err monit[1329]: 'usb1' unable to read filesystem '/media/usb1' state
Nov 11 16:22:55 de838cd8-raspberrymatic user.info monit[1329]: 'usb1' trying to restart
Nov 11 16:23:00 de838cd8-raspberrymatic local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00041F2995E4F5:0","CARRIER_SENSE_LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: CARRIER_SENSE_LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 11 16:23:00 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00041F2995E4F5:0","CARRIER_SENSE_LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 11 16:23:00 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 11 16:23:11 de838cd8-raspberrymatic user.warn monit[1329]: 'hasUSB' status failed (1) -- no output
Nov 11 16:23:11 de838cd8-raspberrymatic user.warn monit[1329]: 'rpi4usb3Check' status failed (1) -- no output
Nov 11 16:23:11 de838cd8-raspberrymatic user.err monit[1329]: Filesystem '/media/usb1' not mounted
Nov 11 16:23:11 de838cd8-raspberrymatic user.err monit[1329]: 'usb1' unable to read filesystem '/media/usb1' state
Nov 11 16:23:11 de838cd8-raspberrymatic user.info monit[1329]: 'usb1' trying to restart
Nov 11 16:23:28 de838cd8-raspberrymatic user.warn monit[1329]: 'hasUSB' status failed (1) -- no output
Nov 11 16:23:28 de838cd8-raspberrymatic user.err monit[1329]: 'rpi4usb3Check' status failed (1) -- no output
Nov 11 16:23:28 de838cd8-raspberrymatic user.err monit[1329]: Filesystem '/media/usb1' not mounted
Nov 11 16:23:28 de838cd8-raspberrymatic user.err monit[1329]: 'usb1' unable to read filesystem '/media/usb1' state
Nov 11 16:23:28 de838cd8-raspberrymatic user.info monit[1329]: 'usb1' trying to restart
Nov 11 16:23:44 de838cd8-raspberrymatic user.warn monit[1329]: 'hasUSB' status failed (1) -- no output
Nov 11 16:23:45 de838cd8-raspberrymatic user.err monit[1329]: Filesystem '/media/usb1' not mounted
Nov 11 16:23:45 de838cd8-raspberrymatic user.err monit[1329]: 'usb1' unable to read filesystem '/media/usb1' state
Nov 11 16:23:45 de838cd8-raspberrymatic user.info monit[1329]: 'usb1' trying to restart
Nov 11 16:24:01 de838cd8-raspberrymatic user.err monit[1329]: 'hasUSB' status failed (1) -- no output
Nov 11 16:24:48 de838cd8-raspberrymatic local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00041F2995E4F5:0","DUTY_CYCLE_LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: DUTY_CYCLE_LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 11 16:24:48 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00041F2995E4F5:0","DUTY_CYCLE_LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 11 16:24:48 de838cd8-raspberrymatic local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]


***** hmserver.log *****
2022-11-11 16:22:16,830 de.eq3.cbcs.server.core.vertx.KeyServerWorker ERROR [vert.x-worker-thread-4] Missing key server configuration parameter (Network.Key) for  mode: KEYSERVER_LOCAL

**Reply by jens-maus** [reply]
Die alte CCU3 mit der selben Config hast du definitiv aus? Und wenn du jetzt beim Pi4 ein HmIP-RFUSB mit der alten Config nutzt musst du natürlich ein paar minuten/stunden warten bis er für alle deine HmIP Geräte eine Neuberechnung der Keys gemacht hat und dazu braucht es internetzugriff von RaspberryMatic aus. Komisch nur das man dazu nix im hmserver.log findet.

Geht der HmIP-RFUSB denn ohne das einspielen deiner alten CCU3 Config? Quasi auf einem frischen system mit einem werkresetteten HmIP Gerät?

**Reply by Mig287** [reply]
Die Logs sind mit einer frischen Installation gemacht. 
Ich habe 3 verschiedene Thermostate Werksresetet und mit einer frischen RaspberryMatic Installation ausprobiert. 
Auch dann bekomme ich keine Verbindung mit irgendeinem Gerät. 

Frisch heißt: deinstalliert, neu start, neu installiert 

Ich finde sehr seltsam, dass ich den Boost mit der alten Config senden kann aber sonst nicht funktioniert. 
edit: die alte CCU3 liegt stromlos neben mir, ich könnte die natürlich wieder anschließen und alle ablernen, aber das sollte ja auch durch den Werksreset erledigt sein.

**Reply by jens-maus** [reply]
Dann spricht das leider immer noch nach funkproblemen. Vllt das RFUSB Defekt bzw. die Antenne nicht sauber verlötet oder kaputt gelötet?!? Oder doch ein Pi4 Problem. Dann mal testweise den Pi3 aus der CCU rausnehmen und damit probieren.

**Reply by Mig287** [reply]
Was da issn Pi3 drinne? 
Ich werde mal googeln wie man da HomeAssistant drauf bekommt, melde mich danach.

**Reply by jens-maus** [reply]
Klar ist da ne Pi3 drin! Sollte man wissen ;)

Und ja da läuft HA auch drauf. Einfach aufschrauben, SD Karte raus, frische SD Karte mit HA rein, fertig!

**Reply by Mig287** [reply]
Ich muss mich nochmal bedanken dafür, dass du dir die Zeit nimmst zum helfen!
Ich habe den Pi aus der CCU rausgeholt und ein HA mit einer anderen SD Karte draufgemacht. 
Habe irgendwie ein wenig gestrauchelt RaspberryMatic zu installieren, das fühlt sich aber schwer nach User Fehler an. 
Aber jetzt am Pi3 mit freshem HA und frischem RaspberryMatic kann ich keine Geräte anlernen..... 

Imho kann es nur noch der Stick sein, wobei dieser es ja zumindest schafft den Boost Befehl zu geben....

PS: ja, der PI4 ist Stromlos ;-)

**Reply by jens-maus** [reply]
Na wenn du doch nun eine CCU3 samt RPI-RF-MOD Funkmodul hast, warum willst du dann überhaupt den HmIP-RFUSB Stick nutzen? Lass doch das RPI-RF-MOD auf dem GPIO Bus, installieren HA auf dem RaspberryPi3 und nutze das RPi-RF-MOD Funkmodul dann zusammen mit HomeAssistant.

**Reply by Mig287** [reply]
Ich habe darüber nachgedacht ob das nicht auch funktioniert...
Ist das auch Plug n play wie mit dem Stick oder muss ich da einer noch zu findenen Anleitung folgen?

**Reply by jens-maus** [reply]
Einfach die Dokumentation studieren:
https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant
